xarray/core/merge.py

```diff
@@ -320,7 +320,7 @@ def merge_coordinates_without_align(
 
 
 def determine_coords(
-    list_of_mappings: Iterable["DatasetLike"]
+    list_of_mappings: Iterable["DatasetLike"],
```

Comment:
We should run ``black --t py36`` to fix all of these at once. Otherwise they'll continue randomly cropping every time a module loses compatibility with py35 for unrelated reasons.

Reply:
I think I've done this—do you get any changes on running that?

Reply:
mmh no, it looks already clean :+1:

Reply:
Cheers!